test/client/util.test.js

```diff
@@ -0,0 +1,129 @@
+describe('The utility module', function() {
+  // TODO throw expects throughout?
+
+  it('performs exponentiation in GF256 correctly.', function() {
```

Comment:
I don't think it makes much sense to test these except for code coverage purposes. I guess the error cases could be tested, but really the only thing to test would be that they return what we expect them to and since they are hard coded to return what we expect, the tests would be something like exp3.get(x) == whatever the hardcoded number in the x index of exp3[] is and that just seems like silly needless testing to me. I wouldn't bother writing tests for something like this in a workplace except maybe the error conditions. If I did test the error conditions I would check 0, -1, 256, and test 1 or 2 values in the appropriate range.